doc/examples/tutorial_null_distributions.md

```diff
@@ -0,0 +1,286 @@
+fig.text(0.09, 0.12, "auto\ncorrelation", rotation=90, ha="center")
+plt.show()
+```
+Each of these methods seem to have different effects:
+- **Jittering**: does exactly that, the jitter in time also results in a jitter in position. 
+  We set the maximal jitter to 2 seconds, which likely matches the timescale of the movements of the animal.
+  The tuning curve and autocorrelation have not changed a lot, meaning we have not broken the relation between spiking and behavior. 
+  Thus, jittering might not be a good randomization method in this case, unless we increase the maximal jitter.
+- **Resampling**: gives us a uniform spread in position, the relation between spiking and behaviour is gone entirely.
+- **Shifting**: has the advantage of keeping the temporal relations between spikes, while only breaking the relation with position.
+- **Shuffling interspike intervals**: looks similar to shifting, it keeps the overall interspike interval distribution.
+
+***
+Null distributions
+------------------
+We will opt for random shifts.
+For each neuron, we will generate `N` pseudo-neurons and compute the mutual/spatial information.
+The resulting distributions of values are what we call the null distributions.
+```{code-cell} ipython3
+N = 500
+shuffles = [nap.shift_timestamps(units, min_shift=20.0) for _ in range(N)]
+
+null_distributions = np.stack(
+    [
+        nap.compute_mutual_information(
+            nap.compute_tuning_curves(
+                data=shuffle,
+                features=position,
+                range=[(-50, 50), (-50, 50)],
+                bins=40,
+            )
+        )["bits/spike"].values
+        for shuffle in shuffles
+    ],
+    axis=1,
+)
+null_distributions.shape
+```
+
+***
+Testing
+------------------
+Ultimately, we will apply the test.
+From a statistical view, we see the null distribution as an empirical distribution of spatial information values that would be expected if the neuron’s firing were unrelated to spatial position.
+We want to determine how unlikely it is that the observed spatial information arose from our null distribution.
+
+Given a significance level, α, we assess whether the observed value lies in the extreme tail of the null distribution.
+If the probability (the p-value) of obtaining a spatial information value **as large as or larger than** the observed one is less than α, we reject the null hypothesis.
+Since we have empirical distributions, we use the percentile the approximate that probability/p-value.
+
+```{code-cell} ipython3
+alpha = 0.01
+thresholds = np.nanpercentile(null_distributions, (1-alpha)*100, axis=1)
+spatial_units = tuning_curves.coords["unit"][spatial_information["bits/spike"].values > thresholds]
+
+g = tuning_curves_smooth.plot(col="unit")
+for ax, (unit_idx, unit_id) in zip(g.axs.flat, enumerate(units)):
+    null = null_distributions[unit_idx]
+    score = spatial_information['bits/spike'][unit_idx]
+    pval = 1- np.sum(null <= score) / len(null)
```

Comment:
I would give a little bit of information here. Like some equations above to explain the p-value